internal/store/postgres/utils.go

```diff
@@ -25,9 +25,13 @@ func addOrderByClause(db *gorm.DB, conditions []string, options addOrderByClause
 		} else {
 			columnOrder := strings.Split(orderBy, ":")
 			column := columnOrder[0]
-			if utils.ContainsString([]string{"updated_at", "created_at"}, column) {
+			if utils.ContainsString(append([]string{"updated_at", "created_at"}, orderByOptions...), column) {
 				if len(columnOrder) == 1 {
-					orderByClauses = append(orderByClauses, fmt.Sprintf(`"%s"`, column))
+					if column == "name" {
+						orderByClauses = append(orderByClauses, fmt.Sprintf(`(CASE WHEN lower("%s") = 'gofood' THEN 1 ELSE 2 END)`, column))
```

Comment:
we can't hardcode `gofood` here since that is internal context.

Reply:
oh, missed this, should be equal to `q`

Reply:
updated